packages/graphql/src/graphql/schema.graphql

```diff
@@ -35,6 +41,26 @@ A date-time string at UTC, such as 2007-12-03T10:15:30Z, compliant with the `dat
 """
 scalar DateTime
 
+type FreeTextQuestionData implements QuestionData {
+  content: String!
+  contentPlain: String!
+  id: Int
+  isArchived: Boolean!
+  isDeleted: Boolean!
+  name: String!
+  options: FreeTextQuestionOptions!
+  type: String!
+}
+
+type FreeTextQuestionOptions {
```

Comment:
These different object types define what exactly is in the options of each question type, and will be used based on the type propertiy of the question (match the corresponding fragment).